modules/rostests/apitests/advapi32/RegCreateKeyExW.c

```diff
@@ -0,0 +1,158 @@
+
```

Comment:
Can you please add our standard copyright header (as mentioned in https://reactos.org/wiki/Coding_Style)?

Reply:
Done. Thanks.